samples/wasi-threads/README.md

```diff
@@ -1 +1,24 @@
-# "WASI threads" sample introduction
\ No newline at end of file
+# "WASI threads" sample introduction
+
+Currently, since the `wasi-sdk` does not have thread support in the latest release, make sure to have [wasi-libc](https://github.com/WebAssembly/wasi-libc) installed. Build it with threads enabled, e.g.
+
+```shell
+make \
+    AR=wasm-micro-runtime/core/deps/llvm/build-lldb/bin/llvm-ar \
+    NM=wasm-micro-runtime/core/deps/llvm/build-lldb/bin/llvm-nm \
+    CC=/opt/wasi-sdk/bin/clang \
```

Comment:
Since we use clang from wasi-sdk, can we also for consistency suggest using the other tools from wasi-sdk?